style/testing/avoid_let_spec.rb

```diff
@@ -0,0 +1,38 @@
+describe SomeClass do
+  describe "#let" do
+    let(:user) { User.new(name: "John") }
+
+    context "avoid using" do
+      it "uses lets" do
+        # BAD
```

Comment:
Here I would put the wording about extracting a helper method in order to customize the thing that's being shared across examples.